README.md

```diff
@@ -9,16 +9,17 @@ may well be used to fire requests to other services.
 
 
 * System dependencies
-    * none
+    * postgres 10.5  -> see setup below
 
 * Configuration
+    you may have to install pstgres gem if you dont already have it 
```

Comment:
typo: pstgres

Reply:
Can you explain why the installation of `pg` from the bundle would not suffice?

Reply:
I will aim to fix the readme changes in one go.  As i was expecting there to be differing views on this.
so as for the second comment around pg, i kept getting issues with bundler complaining the gem doesnt exist maybe there was something else i was doing wrong, we can remove ths iif others who haven't installed postgres gem in any way yet have the same issue